src/analysis/patternSerializer.ts

```diff
@@ -110,7 +113,12 @@ function* _serializeDefinition(def: PatternDefinition) {
   } = def;
 
   if (typeof labelPred === "function") {
-    assert(KNOWN_FUNCTIONS[labelPred.name]);
+    if (
+      KNOWN_FUNCTIONS[labelPred.name] === null ||
+      KNOWN_FUNCTIONS[labelPred.name] === undefined
```

Comment:
could just be `!KNOWN_FUNCTIONS[labelPred.name]`

Reply:
Yeah you are right